cassandra_nodetool/tests/test_unit.py

```diff
@@ -0,0 +1,60 @@
+# (C) Datadog, Inc. 2018
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+from mock import patch
+from os import path
+import common
+from datadog_checks.cassandra_nodetool import CassandraNodetoolCheck
+
+
+def mock_output(*args):
+    with open(path.join(common.HERE, 'fixtures', 'nodetool_output')) as f:
+        contents = f.read()
+        contents = contents.decode('string-escape')
```

Comment:
Do we need this?

Reply:
what do you mean? It is used as a fixture.

Reply:
`contents = contents.decode('string-escape')`